backends/ebpf/runtime/ebpf_map.c

```diff
@@ -43,36 +43,36 @@ static int check_flags(void *elem, unsigned long long map_flags) {
     return EXIT_SUCCESS;
 }
 
-void *bpf_map_lookup_elem(struct bpf_map *map, void *key, unsigned int key_size) {
+void *bpf_map_lookup_elem(struct bpf_map **map, void *key, unsigned int key_size) {
```

Comment:
Why do you need a double pointer?
You never modify the value of map.
Perhaps you can also mark the pointers as const.